modules/contact/test/tests/verification/patch_tests/automatic_patch_update/tests

```diff
@@ -28,6 +28,8 @@
                  "again."
     prereq = sliding_blocks
     min_parallel = 2
+    # max parallel set to avoid not running into error for more procs
```

Comment:
I'm not really sure what this means. It errors with more procs? Is that a problem? What is the error?

Reply:
It stops erroring with more processors. This issue only appears for some decompositions. Since we are checking an error, let's only choose the domain decompositions that actually cause such an error. So it's indeed "avoiding not running into errors for more processors."

Reply:
Something like "The error being tested does not occur for larger numbers of processors" would be more clear.

Reply:
Yeah, we can make that change to make the message more clear. However, "The error being tested does not occur for larger numbers of processors" isn't accurate. It may occur for a larger number of processors at some point.